WalletWasabi/Tor/Http/TorHttpClient.cs

```diff
@@ -191,21 +193,16 @@ public async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, boo
 			// Use clearnet HTTP client when Tor is disabled.
 			if (TorSocks5EndPoint is null)
 			{
-				return await ClearnetRequestAsync(request, isolateStream, token).ConfigureAwait(false);
+				return await ClearnetRequestAsync(request, token).ConfigureAwait(false);
```

Comment:
`isolateStream` not used anymore, you can remove it, and the above SendAsync method as well.